packages/eslint-plugin/docs/rules/no-duplicate-type-constituents.md

```diff
@@ -0,0 +1,49 @@
+---
+description: 'Disallow duplicate union/intersection type members.'
+---
+
+> 🛑 This file is source code, not the primary documentation location! 🛑
+>
+> See **https://typescript-eslint.io/rules/no-duplicate-type-constituents** for documentation.
+
+Although TypeScript supports duplicate union and intersection member values, people usually expect members to have unique values within the same intersection and union. Duplicate values make the code redundant and generally reduce readability.
```

Comment:
[Docs] Here and elsewhere in the file - we should be consistent with how we phrase it.

```suggestion
Although TypeScript supports duplicate union and intersection constituents, people usually expect members to have unique values within the same intersection and union. Duplicate values make the code redundant and generally reduce readability.
```